src/providers/http.js

```diff
@@ -23,6 +24,12 @@ module.exports = () => {
         //   https://github.com/axios/axios/issues/907#issuecomment-322054564
         //   https://github.com/axios/axios/issues/907#issuecomment-373988087
         transformResponse: undefined,
+        // WARNING! Don't use axios (v0.18.x) timeout, it causes the same issue creating

```

Comment:
But this is fixed with axios 0.19?

Reply:
Ah yes, looks like it: https://github.com/axios/axios/releases/tag/v0.19.0-beta.1